### DISCUSSION TITLE
Increase Panel Text Font Size

### DISCUSSION THREAD

**Discussion by usmangt** [grafana/grafana#57151] [Dashboards, Panels & Visualizations]
Based on the following [community posts](https://community.grafana.com/t/can-i-change-font-size-in-a-panel/73259), multiple users (also in the past) want to increase the Panel font size e.g. Heading 1 or 2 etc.

However, this is currently not possible because it is just a standard plain text and cannot be applied e.g. using Markdown language etc.

Therefore creating this request.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
